api/src/main/java/edu/cornell/mannlib/vitro/webapp/email/FreemarkerEmailMessage.java

```diff
@@ -182,15 +208,27 @@ public boolean send() {
 			}
 
 			msg.setSentDate(new Date());
-
-			Transport.send(msg);
+			sendMessage(msg);
 			return true;
 		} catch (MessagingException e) {
 			log.error("Failed to send message.", e);
 			return false;
 		}
 	}
 
+	private void sendMessage(MimeMessage msg) {
+		Thread thread = new Thread() {
```

Comment:
Why the asynchronous send?  I think we already have a problem where the GUI claims that emails were sent successfully even if errors occurred.  It seems that this might make that harder to address.

Reply:
Before that if smtp server have some delays 20-120 sec (it is one of greylisting approaches) then web interface will freeze for that period. 
I agree, current notification about successfully sent email is misleading. I guess It should be replaced with something like "The message is going to be sent shortly" and an option to look up the results in notifications journal.

Reply:
Sounds reasonable.  I guess I had never encountered delays that long, but that would certainly be something to avoid.  We should have a better way of verifying that emails were sent, but that seems out of scope for this PR.